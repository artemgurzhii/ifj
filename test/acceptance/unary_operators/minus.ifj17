Scope
dim a as integer
dim b as integer

a = -3
b = -8.7

End Scope
