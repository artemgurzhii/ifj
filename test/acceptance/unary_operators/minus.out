JUMP Scope
LABEL Scope
DEFVAR GF@a
DEFVAR GF@b
MOVE GF@a int@-3
MOVE GF@b int@-8.7
